internal/api_rest.go

```diff
@@ -0,0 +1,60 @@
+package internal
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+)
+
+var router = chi.NewRouter()
+
+func Router() {
+	if router == nil {
+		router.Get("/Plato/{Nombre}", getPlato)
+		router.Put("/Plato/{Nombre}", putPlato)
+		router.Delete("/Plato/{Nombre}", deletePlato)
+
+		router.Get("/Comedor/{Nombre}", getComedor)
+		router.Put("/Comedor/{Nombre}", putComedor)
+		router.Delete("/Comedor/{Nombre}", deleteComedor)
+
+		router.Get("/Menu/{Fecha}", getMenu)
+		router.Put("/Menu/{Fecha}", putMenu)
+		router.Delete("/Menu/{Fecha}", deleteMenu)
```

Comment:
Te recomiendo que sigas las buenas prácticas:
- https://blog.restcase.com/7-rules-for-rest-api-uri-design/
- https://www.theserverside.com/video/Top-REST-API-URL-naming-convention-standards

Reply:
He puesto las url en minúsculas, si no se me escapa nada creo que queda solucionada la cuestión en [502aa48](https://github.com/antoniorr02/MenuConsulter/pull/40/commits/502aa482b352b603b01aa1547a71dfe8d079e147)